src/libs/Network/index.ts

```diff
@@ -50,7 +45,7 @@ function post(command, data = {}, type = CONST.NETWORK.METHOD.POST, shouldUseSec
         // This check is mainly used to prevent API commands from triggering calls to MainQueue.process() from inside the context of a previous
         // call to MainQueue.process() e.g. calling a Log command without this would cause the requests in mainQueue to double process
         // since we call Log inside MainQueue.process().
-        const shouldProcessImmediately = lodashGet(request, 'data.shouldProcessImmediately', true);
+        const shouldProcessImmediately = request?.data?.shouldProcessImmediately ?? true;
```

Comment:
No need for the `?`s, no?

Reply:
Same https://github.com/Expensify/App/pull/28031#discussion_r1345579630